src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -200,6 +200,13 @@ namespace OpenMS
     params.setValue("EMGScoring:max_iteration", param_.getValue("EMGScoring:max_iteration"));
     params.setValue("EMGScoring:init_mom", param_.getValue("EMGScoring:init_mom"));
     params.setValue("Scores:use_rt_score", "false"); // RT may not be reliable
+    params.setValue("Scores:use_ionseries_scores", "false"); // since FFID only uses MS1 spectra, this is useless
+    params.setValue("Scores:use_ms1_correlation", "false"); // this would be redundant to the "MS2" correlation and since
+    // precursor transition = first product transition, additionally biased
+    params.setValue("Scores:use_ms1_mi", "false"); // same as above. On MS1 level we basically only care about the "MS1 fullscan" scores
+    //TODO for MS1 level scoring there is an additional parameter add_up_spectra with which we can add up spectra around the apex,
+    // to complete ion ladders (and make this score more robust).
```

Comment:
don't know from comment what is meant here. Complete ladders by merging MS1?

Reply:
Ah.. oh.. should be isotopic envelopes of course.